woocommerce/woocommerce
# LYS: Add coming soon page editor links

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #46067.

This PR adds the coming soon page editor links to the WooCommerce homescreen badge and site visibility settings.


![Image](https://github.com/woocommerce/woocommerce/assets/9312929/efa2d5bd-fdae-4ab6-b661-c08fbc40887f)

![Image](https://github.com/woocommerce/woocommerce/assets/9312929/913a30ef-1af3-4bae-a534-5641efcdb955)




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Please exclude from Solaris and GlobalStep testing as this is gated behind a feature flag and is part of the Launch Your Store feature which will be tested as a complete feature when the call for testing is released.


1. Create a new JN site with this branch.
2. When you're redirected to the core profiler, do not proceed.
3. Open a new window and enable `launch-your-store` feature flag.
4. Refresh the core profiler page and skip it.
5. Go to `WooCommerce > Settings`
6. Go to `Site visibility` settings.
7. Click on the coming soon page editor link. (`Your site is hidden from visitors behind a “Coming soon” landing page until it’s ready for viewing. You can customize your “Coming soon” landing page via the **Editor**.`)
8. Verify that the link opens the coming soon page editor.
9. Go to `WooCommerce > Home`
10. Click on the `Site coming soon` badge.
11. Click on the `Customize "Coming soon" page` menu item.
12. Verify that the link opens the coming soon page editor.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
